### ISSUE TITLE
ExceptionInInitializerError in org.emdev.utils.FileUtils.getAbsolutePath

### ISSUE THREAD

**Issue by dschuermann** [SufficientlySecure/document-viewer#231]
From Google Play:
most reported exception on G Play. Happens on many devices with Android 6/7 when opening specific files.

```
java.lang.ExceptionInInitializerError
	at org.emdev.utils.FileUtils.getAbsolutePath(FileUtils.java:66)
	at org.emdev.BaseDroidApp.init(BaseDroidApp.java:90)
	at org.emdev.BaseDroidApp.onCreate(BaseDroidApp.java:56)
	at org.ebookdroid.EBookDroidApp.onCreate(EBookDroidApp.java:46)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1046)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5402)
	at android.app.ActivityThread.-wrap2(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1541)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6123)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:757)
Caused by: java.lang.NullPointerException: Attempt to get length of null array
	at org.emdev.utils.FileUtils.<clinit>(FileUtils.java:27)
	... 14 more
```

**Comment by ericwa** [comment]
Thanks. I think this was fixed by 704ab7fc7d8f40e722f712cfcead1b43230748e8, which wasn't in the google play version until now.

**Comment by dschuermann** [comment]
A great! 😎
